### PR TITLE
Mention the use of dconf to fix errors with gtk themes, since gsettings doesn't work and is what the sway devs suggest

### PR DESCRIPTION
**Pull request by Kommynct**
Literally spent hours trying to use gsettings like sway suggests before I found it gsettings just doesn't work properly on nixos and you need to use dconf.

If this was written here it would've saved me so much time.

### CODE REVIEW

File: pages/Nix/Hyprland on NixOS.md [hunk 1]
```diff
@@ -146,3 +146,11 @@ in {
 
 If your themes for mouse cursors, icons or windows don't load correctly, see the
 relevant section in [Hyprland on Home Manager](../Hyprland-on-Home-Manager).
+
+If you prefer not to use home manager, you can also resolve the issues with GTK themes using dconf like so:
```
Comment (fufexan) [inline pages/Nix/Hyprland on NixOS.md:150]:
```suggestion
If you prefer not to use Home Manager, you can also resolve the issues with GTK themes using dconf like so:
```

File: pages/Nix/Hyprland on NixOS.md [hunk 2]
```diff
@@ -146,3 +146,11 @@ in {
...
+
+exec-once = dconf write /org/gnome/desktop/interface/gtk-theme "'Adwaita'"
+exec-once = dconf write /org/gnome/desktop/interface/icon-theme "'Flat-Remix-Red-Dark'"
+exec-once = dconf write /org/gnome/desktop/interface/document-font-name "'Noto Sans Medium 11'"
+exec-once = dconf write /org/gnome/desktop/interface/font-name "'Noto Sans Medium 11'"
+exec-once = dconf write /org/gnome/desktop/interface/monospace-font-name "'Noto Sans Mono Medium 11'"
```
Comment (fufexan) [inline pages/Nix/Hyprland on NixOS.md:156]:
```suggestion
\`\`\`ini
exec-once = dconf write /org/gnome/desktop/interface/gtk-theme "'Adwaita'"
exec-once = dconf write /org/gnome/desktop/interface/icon-theme "'Flat-Remix-Red-Dark'"
exec-once = dconf write /org/gnome/desktop/interface/document-font-name "'Noto Sans Medium 11'"
exec-once = dconf write /org/gnome/desktop/interface/font-name "'Noto Sans Medium 11'"
exec-once = dconf write /org/gnome/desktop/interface/monospace-font-name "'Noto Sans Mono Medium 11'"
\`\`\`
```

The `\` are there to escape the actual ```

**Review by fufexan — COMMENTED** [review]
dconf keeps its state through reboots, so adding this to `exec-once` is perhaps redundant, as the commands only need to be run when actually changing the values.

Still, I don't know of a better way to do this except activation scripts, which is another beast. So let's leave it at this.

**Comment by Kommynct** [comment]
Sounds good, thanks!

File: pages/Nix/Hyprland on NixOS.md [hunk 3]
```diff
@@ -146,3 +146,13 @@ in {
 
 If your themes for mouse cursors, icons or windows don't load correctly, see the
 relevant section in [Hyprland on Home Manager](../Hyprland-on-Home-Manager).
+
+If you prefer not to use Home Manager, you can also resolve the issues with GTK themes using dconf like so:
+
+\`\`\`ini
```
Comment (fufexan) [inline pages/Nix/Hyprland on NixOS.md:152]: I meant without the actual `\`

Comment (Kommynct) [reply pages/Nix/Hyprland on NixOS.md:152]: I pushed resolve I think that automatically adds your changes, sorry if it does not, I'm not very familiar with github.

**Review by fufexan — APPROVED** [review]

**Pull request merged**